x11-misc/imake/imake-1.0.8-r1.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs xorg-3
+
+DESCRIPTION="C preprocessor interface to the make utility"
+KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~mips ppc ppc64 ~s390 sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
```

Comment:
~arch?

Reply:
It makes more sense but every other packages depend on this new version, want to stabilize it separately first?
I doubt want all these packages to be dropped to ~arch alongside, right?